doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-system-properties-ref.adoc

```diff
@@ -368,4 +368,10 @@ endif::[]
 |`true`, `false`
 |`false`
 |While using JSON marshalling, if the property is set to `true`, it will always return a response in the proper JSON format. For example, if the original response contains only a single number, then the response is wrapped in a JSON format. For example, *{"value" : 1}*.
+
+|`org.kie.server.strict.jaxb.format`
+|`true`, `false`
+|`false`
+|When the property is set to `true`, the {KIE_SERVER} will interpret data in the REST API payload. For example, if a data field is supposed to be a number but it is not a number, you receive an error.
```

Comment:
```suggestion
|When set to `true`, the {KIE_SERVER} interprets the data in the REST API payload. For example, if a data field is supposed to be a number, but it is not a number, you receive an error.
```
I noticed that similar text in this table says "when set to true" and not "When the property is set to true" so I suggest bringing it in line with them. Suggestion also removes passive voice.
"will interpret" - p77 of the style guide. "Write in the simple present tense as much as possible.".

Also, should "interprets" be "validates"?

Reply:
Great comments, Brian. I suggest:
When the value of this property is set to 'true' and do the same for the other one in the table.
It's always KIE_SERVER, not the KIE_SERVER when KIE_SERVER is a noun. You can use 'the' when KIE_SERVER is an adjective. For example, configure the KIE_SERVER instance.

I suggest:
When the value of this property is set to `true`, {KIE_SERVER} validates the data type of the data in the REST API payload. For example, if a data field has the number data type and contains something other than a number, you will receive an error.

Kalyani, confirm this with your SME.

Reply:
@briandooley, @emmurphy1
Thank you so much for the suggestions. Yes, I confirmed this with the SME and acked it.